.npmrc

```diff
@@ -0,0 +1,6 @@
+link-workspace-packages=true
+auto-install-peers=false
+
+# Those needs to be hoisted or the language-server tests will fail. Not sure why
+public-hoist-pattern[]=@vue/*
+public-hoist-pattern[]=svelte2tsx
```

Comment:
I don't really get why this is needed, I ended up finding this because `shamefully-hoist` would fix the issue and I narrowed it down to those packages.

Those packages are depended on in the respective's integrations editor endpoint, and the errors are weird when they're not hoisted, it's almost as if it was loading the wrong version?